mkt/constants/iarc_mappings.py

```diff
@@ -189,15 +191,153 @@
     },
 }
 
-# Change {body: {'key': 'val'}} to {'val': 'key'}.
-REVERSE_DESCS_BY_BODY = [
-    dict([(unicode(v), unicode(k)) for k, v in body_mapping.iteritems()])
-    for body, body_mapping in DESCS.iteritems()]
+# WARNING: When adding a new rating descriptor here also include a migration.
+#          All descriptor keys must be prefixed by the rating body (e.g. USK_).
+#
+# Together with DESCS above these are used to dynamically generate the field
+# list for the RatingDescriptors Django model in mkt.webapps.models.
+# (The model has both v1 and v2 keys, and it's fine, the extra keys are
+#  simply irrelevant depending on the version).
+DESCS_V2 = {
```

Comment:
This is unfortunately not the case. There are descriptors removed from v1. See https://bugzilla.mozilla.org/show_bug.cgi?id=1228364#c1